src/_stylesheets/_inform-inspire.scss

```diff
@@ -1,80 +1,135 @@
+$border-width: 2px;
+$arrow-width: 12px;
+$bottom-position: -5px;
+
 .app-inform-inspire {
-  $padding-left: govuk-spacing(6);
-  $border-width: 2px;
-  $gap: $padding-left;
-  $arrow-width: 16px;
-  margin-top: govuk-spacing(1);
-  margin-bottom: govuk-spacing(4);
-  padding-left: $padding-left;
+  @include govuk-responsive-margin(6, $direction: 'bottom');
+  @include govuk-responsive-padding(6);
+  background-color: $app-light-blue;
+}
+
+// Reset margins in content for non-component elements eg: figures
+.app-inform-inspire__content > * {
+  margin-bottom: 0;
+}
+
+.app-inform-inspire__content--flex {
+  display: flex;
+}
+
+// These modifiers could be looped over, however since there's only the need for
+// 2 cases: 2 items and 5 items, and we don't know what we'd want the gaps to be
+// for other cases, nor can we programmatically determine them right now, we
+// manually generate only these 2.
+.app-inform-inspire__content--flex-2-items {
+  gap: govuk-spacing(5);
+
+  & > * {
+    width: 50%;
+  }
+}
+
+.app-inform-inspire__content--flex-5-items {
+  gap: govuk-spacing(1);
 
-  border-left: solid $border-width;
+  & > * {
+    width: 20%;
+  }
+}
+
+// Overriding .govuk-list's spacing so that it doesn't add extra space at the
+// bottom of the block and creates space between the content element above
+.app-inform-inspire__list {
+  @include govuk-responsive-margin(4, $direction: 'top');
+  margin-bottom: 0;
+}
+
+// Scale bar elements on the homepage
+.app-inform-inspire__bar {
```

Comment:
This probably needs some added styling to be visible in forced colours mode. Currently there's no visible borders or background colour, so the bar graph appearance is lost.